### PR TITLE
UI for Audit Events

### PR DESCRIPTION
**Pull request by ticky**

### CODE REVIEW

File: app/components/organization/SettingsMenu.js [hunk 1]
```diff
@@ -101,6 +101,18 @@ class SettingsMenu extends React.Component {
         )
       },
       {
+        allowed: "teamAdmin",
```
Comment (keithpitt) [inline app/components/organization/SettingsMenu.js]: Hrm... I don't think `teamAdmin` is appropriate for this one (since that'll allow non-org admins to view logs). We'll need to add a new permission, but we can do that in another PR.

So for now, let's use `agentTokenView`

Comment (keithpitt) [inline app/components/organization/SettingsMenu.js]: `agentTokenView` is scoped to organization administrators

Comment (sj26) [inline app/components/organization/SettingsMenu.js]: @ticky we have [a `see_account_audit_events` permission](https://github.com/buildkite/buildkite/blob/audit-ui-permission/app/models/permission.rb#L190-L192), maybe we need to [expose it](https://github.com/buildkite/buildkite/compare/master...audit-ui-permission) to use it here?

File: app/components/audit_log/row.js [hunk 2]
```diff
@@ -0,0 +1,279 @@
...
+      .split(/\s+/);
+
+    const eventVerb = eventTypeSplit.pop().toLowerCase();
+
+    const eventType = eventTypeSplit.join(' ');
+
+    let subjectName = subject.name;
+
+    if (eventTypeName === 'AuditOrganizationCreatedEvent') {
+      subjectName = `${subjectName} 🎉`;
```
Comment (sj26) [inline app/components/audit_log/row.js]: Cute!
